src/components/gamesList.js

```diff
@@ -363,24 +363,23 @@ function AfterJoinRequestSent({ game }) {
     console.log(game);
     return (
         <>
-            <Box>
+            <Box m={2} textAlign='left'>
                 <Text>You requested to join {game.host_info.first_name}'s game at{" "}
                 {game.location_info.park_name} on{" "}
                 {DateTime.fromISO(game.date).toLocaleString({
                     month: "long",
                     day: "numeric",
-                })}{" "}
+                })}<sup>th</sup>{" "}
```

Comment:
is this putting the "th" in 4th? That's so cool! How does that work for 1st and 2nd?